src/components/ActivityCard/ActivityCard.elements.js

```diff
@@ -4,39 +4,56 @@ import device from "../../assets/breakpoints";
 
 export const CardWrapper = styled.div`
   display: flex;
+  margin-right: 1.1vh;
   flex-direction: column;
   /* border: black solid 1px; */
   border-radius: 10px;
   background-color: white;
-
-  width: 210px;
-  height: 360px;
+  min-width: 153px;
+  max-width: 153px;
+  min-height: 360px;
 
   @media ${device.deviceM} {
-    width: 250px;
+    min-width: 222px;
+    max-width: 222px;
     height: 410px;
   }
   @media ${device.deviceL} {
-    max-width: 280px;
+    min-width: 263px;
+    max-width: 263px;
     height: 410px;
   }
   @media ${device.deviceXL} {
-    max-width: 335px;
-    height: 40px;
+    min-width: 194px;
+    max-width: 194px;
+    height: 410px;
   }
 `;
 
-export const Img = styled.img`
-  padding: 1.5vh;
+export const Img = styled.div`
   width: 100%;
-  border-radius: 10%;
+  height: 204px;
+  border-radius: 5%;
   margin: 0px;
+  background-image: url(${(props) => props.img});
+  background-size: cover;
+
+  @media ${device.deviceM} {
+    width: 100%;
+    height: 295px;
+  }
+  @media ${device.deviceL} {
+    height: 320px;
+  }
+  @media ${device.deviceXL} {
+    height: 260px;
+  }
 `;
 
 export const TextWrapper = styled.div`
   display: flex;
   flex-direction: column;
-  padding: 0vh 1.5vh 0.5vh 1.5vh;
+  padding: 0.5vh 0vh 0vh 0vh;
```

Comment:
@deMicale puoi utilizzare direttamente `padding-top` ... In questo caso è meglio utilizzare un'unità di misura assoluta invece di una relativa, quindi meglio i px che cambiano con le media query visto che l'altezza è definita